src/plugins/discover/public/application/angular/doc_table/doc_table.html

```diff
@@ -44,6 +44,7 @@
             index-pattern="indexPattern"
             filter="filter"
             class="kbnDocTable__row"
+            data-test-subj="docTableRow{{ row['$$_isAnchor'] ? ' docTableAnchorRow' : ''}}"
```

Comment:
Nit: `docTableAnchorRow` should not be necessary here, it's just used in context view and there `infinite-scroll` is set to true. It's here added to the paginated doc table that's used in the embeddable.